cea/utilities/solar_equations.py

```diff
@@ -104,21 +104,23 @@ def pyephem(time, latitude, longitude, altitude=0, pressure=101325,
 def calc_sun_properties(latitude, longitude, weather_data, gv):
 
     date = pd.date_range(gv.date_start, periods=8760, freq='H')
+    hour_date = date.hour
```

Comment:
couldn't we just pass the date vector to `get_hour_angle`? then we don't have to worry about `min_date` potentially meaning `min(date)` etc...